Master RMarkdown Document & Render Code/Global Script.R

```diff
@@ -21,6 +21,10 @@ library(glue)
 library(fs)
 library(arrow)
 library(phsstyles)
+library(scales)
+library(flextable)
+library(officer)
+library(magrittr)
```

Comment:
Why is magrittr needed specifically? The standard pipe `%>%` is loaded (re-exported) with most tidyverse packages, dplyr etc. So loading magrittr is only required if you're using one of its other pipes or functions.